service/src/main/java/uk/gov/hmcts/reform/fpl/config/DocumentConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.fpl.config;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Data
+@Configuration
+public class DocumentConfiguration {
```

Comment:
I don't like the name DocumentConfiguration as it doesn't have to do anything with the document - it's the GatewayConfiguration and are configuring the gatewayUrl. 

At least DocumentGateway? but still the use is more broader than just documents.